src/neptune/metadata_containers/project.py

```diff
@@ -199,8 +199,8 @@ def fetch_runs_table(
                 Matching any element of the list is sufficient to pass the criterion.
                 Defaults to `None`.
             state: Run state, or list of states.
-                Example: `"running"` or `["idle", "running"]`.
-                Possible values: "idle", "running".
+                Example: `"Active"` or `["Inactive", "Active"]`.
```

Comment:
Ok but we have to translate it in two places:
- On NQL creation (a query to Leaderboard filters)
- On table entries creation (the result of `fetch_runs_table`)

Reply:
Ok...
So I've created intermediate `RunState` enum which is responsible for translating strings.